### ISSUE TITLE
Multiple functions break because AOI::getBoundingBox is not found

### ISSUE THREAD

**Issue by jsta** [mikejohnson51/HydroData#30]
Looks like it has been replaced by `AOI::bbox_get`

**Comment by mikejohnson51** [comment]
Hi, 

Yes, after a fantastic code review of AOI by @dblodgett-usgs I refined some function names and scope and haven't got around to fixing the packages relying on it. It is on my to-do list and I would welcome any contribution given your great R talents,

Mike

**Comment by jsta** [comment]
In process on this. But I'm stumped on `AOI::aoiProj`. What does that command map to in the current version?

**Comment by mikejohnson51** [comment]
That was a naive (bad) way of storing EPSG 4326 with the corresponding proj4 string. I think they should all be replaceable by 4326.

Thanks!

**Comment by mikejohnson51** [comment]
Sooo... I am super embarrassed now ... I fixed most of these issues and never pushed to GitHub 🤦 The code could still use a good amount of work/review and I hope to eventually write it up as a publication (maybe [EMS](https://www.journals.elsevier.com/environmental-modelling-and-software)) with some compelling use cases. Would you be interested in pushing it in that direction as a collaboration? 

I will get that push up tonight/tomorrow morning...

Again, I feel so bad and hope you didn't spend to much time on this!

**Comment by jsta** [comment]
No problem I didn't spend too much time. Yes, I'd be interested in helping out with a paper effort. My latest interest in the package is the wrapper to NLDI so maybe that's an area where I could contribute a use case..

**Comment by dblodgett-usgs** [comment]
Just lurking here -- @ldecicco-USGS and I talked about moving an NLDI wrapper client from nhdplusTools out to dataRetrieval. Maybe a little collaboration to make sure it's good for all the use cases we have would be in order...

**Comment by mikejohnson51** [comment]
@dblodgett-usgs yeah what you raise is  part of the larger point where I stopped focusing on this package - with the intention of coming back... The idea of a single data aggregator was getting to be heavy on maintenance and better things were being done in packages like `nhdplusTools`.  Mostly I was waiting for dust to settle :)

That said, perhaps part of our next talk can be figuring out where nhd "discovery" tools belong. Some really useful ones sit in `nhdplusTools`, here, and in `nwmHistoric`. Knowing where they go and consolidating them would be great!

The one aspect I like about this wrapper is that it builds out many NLDI requests and returns a list of spatial (sf) objects. I also like the generalization (in a naive sense) of the feature types (e.g. just the USGS siteID  rather then "USGS-#")
